include/cutlass/gemm/collective/builders/pvc_mma_builder.inl

```diff
@@ -0,0 +1,125 @@
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ **************************************************************************************************/
+
+#pragma once
+
+#include <cutlass/arch/arch.h>
+#include <cutlass/gemm/dispatch_policy.hpp>
+
+#include "cutlass/gemm/collective/collective_mma.hpp"
+
+namespace cutlass::gemm::collective {
+
+  // Intel PVC 3 stage dpas pipeline, using prefetch
+  // Also the auto builder
+
+template <
+  class ElementA,
+  class GmemLayoutATag,
+  int AlignmentA,
+  class ElementB,
+  class GmemLayoutBTag,
+  int AlignmentB,
+  class ElementAccumulator,
+  class TileShape_MNK,
+  class KernelScheduleType
+  > 
+struct CollectiveBuilder<
+  arch::IntelPVC,
+  arch::OpClassTensorOp,   // Reusing opClassTensorOp instead of declaring a new "OpClassDpasOP" as they have the same meaning
+  ElementA,
+  GmemLayoutATag,
+  AlignmentA,
+  ElementB,
+  GmemLayoutBTag,
+  AlignmentB,
+  ElementAccumulator,
+  TileShape_MNK,
+  Shape<_1, _1, _1>,    // Cluster Shape, which has no meaning in PVC
+  cutlass::gemm::collective::StageCountAuto, // Stage Count needn't be a parameter
+  KernelScheduleType,
+  cute::enable_if_t<
+    (cute::is_same_v<KernelScheduleType, KernelPVC> ||
+     cute::is_same_v<KernelScheduleType, KernelScheduleAuto>) &&  
+    cute::is_same_v<GmemLayoutATag, cutlass::layout::RowMajor> && // Different struct specialization because this will change copy atoms
+    cute::is_same_v<GmemLayoutBTag, cutlass::layout::RowMajor>
+  >
+    >{
+
+      #ifdef SYCL_NVIDIA_TARGET
+        static_assert(cutlass::detail::dependent_false<arch::IntelPVC>, 
+          "Trying to use PVC pipeline on Non PVC hardware");
+      #endif
+      static_assert(is_static<TileShape_MNK>::value);
+      static_assert(cute::is_same_v<ElementA, bfloat16_t>, "PVC single stage pipeline requires ElementA to be of type bfloat16_t");
```

Comment:
```suggestion
      static_assert(cute::is_same_v<ElementA, bfloat16_t>, "PVC multi-stage pipeline requires ElementA to be of type bfloat16_t");
```

Reply:
I will changing all mentions of PVC to Intel Here

Reply:
DOne